### PR TITLE
file-systems/unixfs: fix default leaf format.

### PR DESCRIPTION
**Pull request by raulk**

### CODE REVIEW

File: docs/concepts/file-systems.md [hunk 1]
```diff
@@ -285,9 +285,9 @@ When an object is added to IPFS, it is chunked up into smaller parts, each part
 
 The leaf format takes two format options, UnixFS leaves and raw leaves:
 
-- The UnixFS leaves format adds a data wrapper on newly added objects to produce UnixFS leaves with additional data sizes. This wrapper is used to determine whether newly added objects are files or directories.
+- The UnixFS leaves format (default) adds a data wrapper on newly added objects to produce UnixFS leaves with additional data sizes. This wrapper is used to determine whether newly added objects are files or directories.
```
Comment (lidel) [inline docs/concepts/file-systems.md:288]:
```suggestion
- The UnixFS leaves format (default for CIDv0) adds a data wrapper on newly added objects to produce UnixFS leaves with additional data sizes. This wrapper is used to determine whether newly added objects are files or directories.
```

File: docs/concepts/file-systems.md [hunk 2]
```diff
@@ -285,9 +285,9 @@ When an object is added to IPFS, it is chunked up into smaller parts, each part
...
 
-- The raw leaves format is the default format on IPFS where nodes output from chunking will be raw data from the file with a CID type of 'raw'. This is mainly configured for backward compatibility with formats that used a UnixFS Data object.
+- The raw leaves format (experimental, likely future default) on IPFS where nodes output from chunking will be raw data from the file with a CID type of 'raw'. This is mainly configured for backward compatibility with formats that used a UnixFS Data object.
```
Comment (lidel) [inline docs/concepts/file-systems.md:290]:
```suggestion
- The raw leaves format (default for CIDv1 created with `ipfs add --cid-version 1`, soon to become the global default) on IPFS where nodes output from chunking will be raw data from the file with a CID codec of 'raw'. This is mainly configured for backward compatibility with formats that used a UnixFS Data object.
```

**Review by lidel — COMMENTED** [review]
I think it's useful to note that raw leaves are enabled by default when user opts-in to CIDv1 during the import, and note that will become the default soon. 

Feel free to rephrase below suggestions.

**Comment by raulk** [comment]
Thanks @lidel, done!

**Comment by achingbrain** [comment]
> note that will become the default soon

Is there a timeline for this?  Last time this came up I think @Stebalien had some ideas around introducing 'import profiles' that would apply raw leaves, cidv1, etc, rather than changing the defaults.

**Review by Stebalien — APPROVED** [review]

**Comment by Stebalien** [comment]
> Last time this came up I think @Stebalien had some ideas around introducing 'import profiles' that would apply raw leaves, cidv1, etc, rather than changing the defaults.

Honestly, at this point, I'd just change the default and document it in the release notes. If it's _just_ one flag, that shouldn't be a big issue. As we make more and more changes, we can go back and add a profiles flag.

I'm also happy to do that now, but I don't want to block on that.

File: docs/concepts/file-systems.md [hunk 3]
```diff
@@ -285,9 +285,9 @@ When an object is added to IPFS, it is chunked up into smaller parts, each part
 
 The leaf format takes two format options, UnixFS leaves and raw leaves:
 
-- The UnixFS leaves format adds a data wrapper on newly added objects to produce UnixFS leaves with additional data sizes. This wrapper is used to determine whether newly added objects are files or directories.
+- The UnixFS leaves format (default for CIDv0) adds a data wrapper on newly added objects to produce UnixFS leaves with additional data sizes. This wrapper is used to determine whether newly added objects are files or directories.
```
Comment (johnnymatthews) [inline docs/concepts/file-systems.md:288]:
```suggestion
- The UnixFS leaves format adds a data wrapper on newly added objects to produce UnixFS leaves with additional data sizes. This wrapper is used to determine whether newly added objects are files or directories. This format is the default for CIDv0.
```

File: docs/concepts/file-systems.md [hunk 4]
```diff
@@ -285,9 +285,9 @@ When an object is added to IPFS, it is chunked up into smaller parts, each part
...
 
-- The raw leaves format is the default format on IPFS where nodes output from chunking will be raw data from the file with a CID type of 'raw'. This is mainly configured for backward compatibility with formats that used a UnixFS Data object.
+- The raw leaves format (default for CIDv1 created with `ipfs add --cid-version 1`, soon to become the global default) on IPFS where nodes output from chunking will be raw data from the file with a CID codec of 'raw'. This is mainly configured for backward compatibility with formats that used a UnixFS Data object.
```
Comment (johnnymatthews) [inline docs/concepts/file-systems.md:290]:
```suggestion
- The raw leaves format on IPFS where nodes output from chunking will be raw data from the file with a CID codec of 'raw'. This is mainly configured for backward compatibility with formats that used a UnixFS Data object. This format is the default for CIDv1 created with `ipfs add --cid-version 1`, soon to become the global default.
```

**Review by johnnymatthews — APPROVED** [review]

**Pull request merged**